### PR TITLE
fix(docs): changes(ZEIT to Vercel) in core-utils and recipes

### PR DESCRIPTION
**Pull request by axiiomatic**
changes:
- [x]  Fix - changed ZEIT to Vercel in `packages/gatsby-core-utils/README.md`
- [x]  Fix - changed now to Vercel in ` /docs/recipes/deploying-your-site.md`

## Related Issues
#24019 [Umbrella] Rename Zeit to Vercel

### CODE REVIEW

File: docs/docs/recipes/deploying-your-site.md [hunk 1]
```diff
@@ -95,11 +95,11 @@ Use [Vercel CLI](https://vercel.com/download) to deploy your Gatsby application
 
 ### Directions
 
-1. Login into Now CLI using `now login`
+1. Login into Now CLI using `vercel login`
```
Comment (muescha) [inline docs/docs/recipes/deploying-your-site.md:98]:
```suggestion
1. Login into Vercel CLI using `vercel login`
```

**Comment by muescha** [comment]
i see in line 90 there is a 

```
// {CI_NAME: "ZEIT Now"}
```

but i think this has to be changed in `packages/gatsby-core-utils/src/ci.ts` before

**Comment by muescha** [comment]
still i would suggest to leave it unchanged until it is changed in `core-utils/src/ci.ts`

**Comment by axiiomatic** [comment]
so revert back to ZEIT?

**Comment by muescha** [comment]
> so revert back to ZEIT?

leave it as it is because you not touched line 90

**Comment by axiiomatic** [comment]
@muescha any clue why the tests are failing?

**Comment by muescha** [comment]
I don’t have access to the logs so that I have no clue why tests are failing

**Comment by iamdarshshah** [comment]
We can't make changes in /packages for now as it is dependent on `ci-info`. `ci-info` is still not updated as per the latest changes made by ZEIT to Vercel. Let's wait for `ci-info` to provide Vercel as a `Supported CI tools`.

Then we can change the packages (.ts files) as per the` Officially support for CI server - Vercel` docs. **[ reference: #24019 ]**

**Comment by axiiomatic** [comment]
No problem. happy to help :)

**Pull request merged**